src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -51,7 +52,7 @@ namespace OpenLoco::Audio
     [[maybe_unused]] constexpr int32_t kPlayAtLocation = 0x8001;
     [[maybe_unused]] constexpr int32_t kNumSoundChannels = 16;
 
-    static constexpr uint8_t kNoSong = 0xFF;
+    static constexpr Jukebox::MusicId kNoSong = 0xFF;
```

Comment:
Would it make sense to use `~0` instead of `0xFF` here?

Reply:
Another alternative is `UCHAR_MAX`, though that also wouldn't change if MusicId is redefined.

Reply:
Rather than using C constants, you could use std::limits instead. Not sure it's worth it, though.

Reply:
I would just use the standard 0xFF since that's what the rest of the codebase uses